Makefile

```diff
@@ -171,29 +179,35 @@ misspell-fix: $(TOOLBIN)/misspell
 
 
 ## --------------------------------------
-## Deploying
+## Deploy all (CRDs + Controller)
 ## --------------------------------------
 
-# Install CRDs into a cluster
+# Deploy controller in the configured Kubernetes cluster in ~/.kube/config
+.PHONY: deploy
+deploy: $(TOOLBIN)/kubectl
+	$(TOOLBIN)/kubectl apply -f deploy/kube-app-manager-aio_$(VERSION).yaml
+
+# unDeploy controller in the configured Kubernetes cluster in ~/.kube/config
+.PHONY: undeploy
+undeploy: $(TOOLBIN)/kubectl
+		$(TOOLBIN)/kubectl delete -f deploy/kube-app-manager-aio_$(VERSION).yaml
+
+## --------------------------------------
+## Deploy CRDs only
+## --------------------------------------
+# Install CRDs into a cluster,
 .PHONY: install
-install: $(TOOLBIN)/kustomize $(TOOLBIN)/kubectl
+deploy-crd: $(TOOLBIN)/kustomize $(TOOLBIN)/kubectl
 	$(TOOLBIN)/kustomize build config/crd| $(TOOLBIN)/kubectl apply -f -
```

Comment:
install vs deploy wording is confusing in kubernetes ecosystem . Especially in this case when it's a substep of the whole deployment.
Renamed it in deploy-crd